microservices-connector/cmd/router/main.go

```diff
@@ -477,6 +479,112 @@ func mcGraphHandler(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+func mcDataHandler(w http.ResponseWriter, r *http.Request) {
+	defaultNode := mcGraph.Spec.Nodes[defaultNodeName]
```

Comment:
If there is no dataprep in the GMC config, this handler should return info like " this endpoint is not available" to end user.

Reply:
Yeah, I added the error feedback when there is no dataprep is found.